benchmark/paddle/DQN/README.md

```diff
@@ -0,0 +1,27 @@
+## Reproduce DQN with PARL
+Based on PARL, we provide a simple demonstration of DQN.
```

Comment:
we provide a simple demonstration of DQN.
----------->
we provide a simple example of the DQN algorithm.